LBH.AdultSocialCare.Api/V1/Gateways/Security/Concrete/UsersGateway.cs

```diff
@@ -56,5 +58,26 @@ public async Task<AppUserDomain> GetAsync(Guid userId)
                 .FirstOrDefaultAsync(item => item.Id == userId).ConfigureAwait(false);
             return user?.ToDomain();
         }
+
+        public async Task<IEnumerable<UsersMinimalDomain>> GetUsers(RolesEnum rolesType)
+        {
+            var roleId = await _databaseContext.Roles
+                .Where(r => r.Name.Equals(rolesType.ToString()))
+                .Select(r => r.Id)
+                .FirstOrDefaultAsync();
+
+            return await _databaseContext.Roles
```

Comment:
Furkan, can I write below query like this? 

Did I miss anything with that query?
```suggestion
        _databaseContext.UserRoles.Where(x => x.Role.Name == "aa")
                                      .Select(x => new UsersMinimalDomain
                                      {
                                          Id = x.User.Id,
                                          Name = x.User.Name
                                      });
```